client/components/tinymce/plugins/wpcom-view/views/simple-payments/index.jsx

```diff
@@ -25,14 +25,14 @@ class SimplePaymentsView extends Component {
 			return ( <QuerySimplePayments siteId={ siteId } productId={ productId } /> );
 		}
 
-		const { title, description, price, currency } = product;
+		const { title, description, price, currency, multiple } = product;
+
+		const paypalButtonImageUrl = '/calypso/images/simple-payments/preview-paypal-button-gray.png';
+		const multipleIsEnabled = Number( multiple ) !== 0;
```

Comment:
I think notion of *availability*, a range value, may be more appropriate since P2P commerce is almost always limited in availability.